### PR TITLE
feat(stepper): prefix change + singular capital case

### PR DESCRIPTION
**Pull request by theJohnnyMe**
**Describe pull-request**  
prefix change + singular capital case for stepper component

**Solving issue**  
Fixes: DTS-1648

**How to test**  
1. Go to the Storybook link below
2. Check that the component design is the same as in the main branch
3. Prefix is changed from sdds to tds 
4. in doc files we mentioned "Stepper" instead of stepper

**Suggested test steps**
- [ ] Browser testing (Chrome, Safari, Firefox) 
- [ ] Keyboard operability
- [ ] Interactive elements have labels.
- [ ] Storybook controls
- [ ] Design/controls/props is aligned with other components 
- [ ] Dark/light mode and variants 
- [ ] Input fields – values should be displayed properly 
- [ ] Events

### CODE REVIEW

File: core/src/components/stepper/readme.md [hunk 1]
```diff
@@ -7,20 +7,20 @@
...
+| --------------- | ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------- | --------------------- |
+| `hideLabels`    | `hide-labels`    | Hides the label for the child components if true.                                                                                                                                                                                                 | `boolean`                    | `false`               |
+| `labelPosition` | `label-position` | Text position, only available on a direction: horizontal                                                                                                                                                                                          | `"aside" \| "below"`         | `'below'`             |
```
Comment (mJarsater) [inline core/src/components/stepper/readme.md:13]:
```suggestion
| `labelPosition` | `label-position` | Text position, only available on direction: horizontal                                                                                                                                                                                          | `"aside" \| "below"`         | `'below'`             |
```

File: core/src/components/stepper/stepper-item/stepper-item.tsx
Comment (mJarsater) [inline core/src/components/stepper/stepper-item/stepper-item.tsx:1]: Should this be tds-stepper and tds-step instead? To follow our convention?

Comment (mJarsater) [reply core/src/components/stepper/stepper-item/stepper-item.tsx:1]: `<tds-stepper>
      <tds-step></tds-step>
      <tds-step></tds-step>
</tds-stepper>`

Comment (theJohnnyMe) [reply core/src/components/stepper/stepper-item/stepper-item.tsx:1]: I was not sure either should I change it or not, but let me make it happen and we can confirm with the team.

Comment (theJohnnyMe) [reply core/src/components/stepper/stepper-item/stepper-item.tsx:1]: @mJarsater  - Changes pushed!

**Review by mJarsater — APPROVED** [review]
Really nice!

**Pull request merged**